src/styles/index.css

```diff
@@ -0,0 +1,68 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: lightblue;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+h1 {
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+.product-card {
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  transition: transform 0.3s ease, box-shadow 0.3s ease;
+}
+.product-card:hover {
+  transform: translateY(-5px);
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.1);
+}
+.product-card__image {
+  max-width: 100%;
+  border-radius: 4px;
+  margin-bottom: 12px;
+}
+.product-card__title {
+  font-size: 18px;
+  color: #333;
+  margin: 8px 0;
+}
+.product-card__description {
+  font-size: 14px;
+  color: black;
+  margin-bottom: 12px;
+}
+.product-card__price {
+  font-size: 16px;
+  font-weight: bold;
+  color: #333;
+  margin-bottom: 16px;
+}
+.product-card__button {
+  padding: 8px 16px;
+  background-color: #007bff;
```

Comment:
Consider using SCSS variables for colors like `background-color` to make it easier to manage and update these values.